libs/api/utils/src/errorList.ts

```diff
@@ -50,4 +50,8 @@ export const ERROR_LIST = {
     code: 401,
     message: 'Organisation API key has been expired.',
   },
+  USER_CREDIT_CAN_NOT_BE_NEGATIVE: {
+    code: 500,
```

Comment:
**thought:** Is it really our fault (error 500), when it's user's credits problem? I think 402 would be better. Or anything from 4xx family.

https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/402